templates/web/src/payload.ts.twig

```diff
@@ -0,0 +1,40 @@
+export class Payload extends Blob {
+    public name: string | null = null;
+
+    constructor(data: BlobPart[], name?: string) {
```

Comment:
Whats difference between Buffer and BlobPart[]? We use Buffer in Node

Reply:
BlobPart can be string, uintarray, arraybuffer ...